pkg/odo/cli/component/common_link.go

```diff
@@ -343,3 +379,22 @@ func (o *commonLinkOptions) waitForLinkToComplete() (err error) {
 	_, err = o.Client.WaitAndGetPod(podSelector, corev1.PodRunning, "Waiting for component to start")
 	return err
 }
+
+// getSvcFullName returns service name in the format <service-type>/<service-name>
+func getSvcFullName(serviceType, serviceName string) string {
+	return strings.Join([]string{serviceType, serviceName}, "/")
+}
+
+// getSBRName creates a name to be used for creation/deletion of SBR during link/unlink operations
+func getSBRName(componentName, serviceType, serviceName string) string {
```

Comment:
I am losing track of such helper functions, are we sure we are reusing them?

Reply:
As discussed on call, I create a helper only when it's used at least twice.